requirements.txt

```diff
@@ -1,14 +1,13 @@
 # PyPI dependencies
 accept-types==0.4.1
-aiohttp-socks==0.5.5
+aiohttp==3.8.1
 aiokafka==0.7.2
+avro==1.11.0
 jsonschema==3.2.0
-lz4==3.0.2
-requests==2.27.1
```

Comment:
Requests library seems to be directly used in tests code, so instead of removing, moving this to requirements-dev.txt?

Reply:
Requests is in the indirect requirements. I'll add it also to in the dev requirements.

Reply:
No, I won't add it back as the `kafka-python` is already in the file: https://github.com/aiven/karapace/pull/448/files#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552R19

The `kafka-python` is installed from a fork.